lua/harpoon/init.lua

```diff
@@ -159,7 +167,14 @@ M.print_config = function()
 end
 
 -- Sets a default config with no values
-M.setup({projects = {}})
+M.setup(
```

Comment:
Lets change setup to be called with nothing 

```
M.setup()
```

Then lets change the defaults inside the function instead of here since they will get wiped out with any call to setup.